networkx/algorithms/shortest_paths/weighted.py

```diff
@@ -2285,9 +2289,15 @@ def bidirectional_dijkstra(G, source, target, weight="weight"):
 
         for w, d in neighs[dir][v].items():
             if dir == 0:  # forward
-                vwLength = dists[dir][v] + weight(v, w, d)
+                cost = weight(v, w, d)
+                if cost is None:
+                    continue
+                vwLength = dists[dir][v] + cost
             else:  # back, must remember to change v,w->w,v
-                vwLength = dists[dir][v] + weight(w, v, d)
+                cost = weight(w, v, d)
+                if cost is None:
+                    continue
+                vwLength = dists[dir][v] + cost
```

Comment:
Nice idea!  I hadn't realized this feature wasn't available.

The code might be more readable by reducing duplication. What do you think of:
```python
    cost = weight(v, w, d) if dir == 0 else weight(w, v, d)
    if cost is None:
        continue
    vwLength = dists[dir][v] + cost
```

Reply:
That should also do the job ;)